packages/pwa-kit-react-sdk/src/ssr/universal/components/with-react-query/index.js

```diff
@@ -75,29 +75,33 @@ export const withReactQuery = (Wrapped, options = {}) => {
             const queryCache = queryClient.getQueryCache()
             const queries = queryCache.getAll().filter((q) => q.options.enabled !== false)
             await Promise.all(
-                queries.map((q, i) => {
+                queries.map(async (q, i) => {
                     // always include the index to avoid duplicate entries
                     const displayName = q.meta?.displayName ? `${q.meta?.displayName}-${i}` : `${i}`
                     res.__performanceTimer.mark(
                         `${PERFORMANCE_MARKS.reactQueryUseQuery}.${displayName}`,
                         'start'
                     )
-                    return q
-                        .fetch()
-                        .then((result) => {
-                            res.__performanceTimer.mark(
-                                `${PERFORMANCE_MARKS.reactQueryUseQuery}.${displayName}`,
-                                'end',
-                                {
-                                    detail: q.queryHash
-                                }
-                            )
-                            return result
-                        })
-                        .catch(() => {
-                            // If there's an error in this fetch, react-query will log the error
-                            // On our end, simply catch any error and move on to the next query
+
+                    try {
+                        const result = await q.fetch()
+                        return result
+                    } catch (err) {
+                        logger.error('Query during SSR results in an error', {
```

Comment:
Looking at the AC is see "logs should have performance keyword". Did we miss adding the "performance" in the log message?